### ISSUE TITLE
Improve readme.md (always open issue, because readme file has a lot of room for improvement :)

### ISSUE THREAD

**Issue by Nirbhayparmar** [Nirbhayparmar/gradientiser-BgGenerator#12]
**Describe the bug**
This project has readme.md file, we but it is just bare minimum, if anyone interested in No code opensource contribution can work on this.

**Comment by Jay179-sudo** [comment]
Hi! I'd like to claim this issue!

**Comment by Srinika99** [comment]
> Hi! I'd like to claim this issue!

Hey, I'd be glad to work on this with you.

**Comment by Munnachienyi** [comment]
hey, I can see you want an improved readme.md file. I would love to improve it. can I work on this issue?

**Comment by Ayush-Khandelwal28** [comment]
Hey, I would like to improve the README.md file.

**Comment by Nirbhayparmar** [comment]
@Jay179-sudo , assigning this issue to you, work on this issue and open a pr.

**Comment by Nirbhayparmar** [comment]
any body else want to improve readme further with update gif and with tags and labels? I am keeping this issue open so anyone can claim it.

**Comment by 12-malak** [comment]
@Nirbhayparmar Hi! I'd like to claim this issue.

**Comment by 12-malak** [comment]
@Nirbhayparmar Created PR #33  . Please review and merge. Also, please feel free to add the hacktoberfest-accepted label once approved.

**Comment by Nirbhayparmar** [comment]
> @Nirbhayparmar Created PR #33 . Please review and merge. Also, please feel free to add the hacktoberfest-accepted label once approved.

Accepted your PR and added hacktoberfest-accepted label on it.

**Comment by amenhasfaw** [comment]
@Nirbhayparmar Hey, I'd like to work on the readme file. Should I start working on it?

**Comment by Nirbhayparmar** [comment]
Yes @amenhasfaw you can work on this issue. You can update the readme file by replacing the demo goif with newer one.

**Comment by amenhasfaw** [comment]
@Nirbhayparmar I've created a PR with the GIF change and some other changes. Please review and merge the PR. Also, please add the hacktoberfest-accepted label if you find my work to be satisfactory.

**Comment by lemostrash** [comment]
@Nirbhayparmar I have some simple ideas for the readme, can I assign them?

**Comment by Nirbhayparmar** [comment]
Ok

**Comment by olybeke** [comment]
hi @Nirbhayparmar I would like to work on the readme file also. can you assign it to me?

**Comment by Nirbhayparmar** [comment]
Okay @olybeke, you can work on it.
